packages/fuchsia_remote_debug_protocol/lib/src/fuchsia_remote_connection.dart

```diff
@@ -506,6 +507,30 @@ class FuchsiaRemoteConnection {
     _pollDartVms = true;
   }
 
+  /// Helper for getDeviceServicePorts() to extract the vm_service_port from
+  /// json response.
+  List<int> getVmServicePortFromInspectSnapshot(dynamic inspectSnapshot) {
+    final ports = <int>[];
+    for (final item in inspectSnapshot as List<dynamic>) {
```

Comment:
again, avoid dynamic, and this can be a more specific type (since we cast later)